src/compiler/scala/tools/nsc/typechecker/Typers.scala

```diff
@@ -5759,13 +5759,15 @@ trait Typers extends Adaptations with Tags with TypersTracking with PatternTyper
 
       // Extract and insert stabilizing ValDefs (if any) which might have been
       // introduced during the typing of the original expression.
-      def insertStabilizer(tree: Tree, original: Tree): Tree =
-        stabilizingDefinitions(original) match {
+      def insertStabilizer(tree: Tree, original: Tree): Tree = {
```

Comment:
Bit of a minor off-topic point: there is no need for outer braces around the `if-then-else`. Is there any convention on that style?

Reply:
At the moment there isn't a convention.